software/sys_controller/tvp7002/video_modes.c

```diff
@@ -163,6 +187,11 @@ alt_8 get_mode_id(alt_u32 totlines, alt_u8 progressive, alt_u32 hz, video_type t
                     cm.fpga_hmultmode = FPGA_H_MULTMODE_OPTIMIZED;
                     cm.sample_mult = 5;
                     break;
+                case MODE_L2_480x272:
+                    cm.fpga_vmultmode = FPGA_V_MULTMODE_2X;
+                    cm.fpga_hmultmode = FPGA_H_MULTMODE_OPTIMIZED;
+                    cm.sample_mult = 2;
+                    break;
```

Comment:
This block could be combined with MODE_L2_512_COL.

Reply:
Done